src/test/java/edu/hm/hafner/grading/QualityGateEvaluationTest.java

```diff
@@ -0,0 +1,115 @@
+package edu.hm.hafner.grading;
+
+import java.util.List;
+import java.util.Map;
+
+import org.junit.jupiter.api.Test;
+
+import edu.hm.hafner.util.FilteredLog;
+
+import static org.assertj.core.api.Assertions.*;
+
+/**
+ * Tests for {@link QualityGateEvaluation}.
+ */
+class QualityGateEvaluationTest {
```

Comment:
Otherwise the tests break on my machine as I am using a German locale that renders numbers with comma.

```suggestion
@DefaultLocale("en")
class QualityGateEvaluationTest {
```